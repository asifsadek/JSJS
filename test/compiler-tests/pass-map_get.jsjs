val a = { "foo" : "bar"};
print(get(a, "foo"));

val b = { 1 : [1, 2, 3] };
print(get(b, 1));

val c = { true : 2, 
          false : 3 };
print(get(c, true));

val d = { 1 : { "foo" : 2}};
print(get(d, 1));
