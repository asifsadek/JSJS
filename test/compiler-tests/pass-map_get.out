bar
List [ 1, 2, 3 ]
2
Map { "foo": 2 }
